### ISSUE TITLE
SQLSTATE[HY000] [2002] Error After Installing Laravel Telescope

### ISSUE THREAD

**Issue by pejeio** [laravel/telescope#1514]
### Telescope Version

5.2.2

### Laravel Version

11.10.0

### PHP Version

8.3.2

### Database Driver & Version

mysql/mysql-server:8.0 image as defined in the docker-compose.yml of sail

### Description

I encountered an error when trying to install Laravel Telescope in my project. After running the command `composer require laravel/telescope --dev` as per the documentation, I noticed the following error in the Laravel log (`/storage/logs/laravel.log`):

```
local.ERROR: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for mysql failed: nodename nor servname provided, or not known (Connection: mysql, SQL: insert into `telescope_entries`...
```

However, when I remove Telescope by running `composer remove laravel/telescope --dev`, the error no longer appears in the `laravel.log`.

Here are some relevant entries from my `.env` file:

```
DB_CONNECTION=mysql
DB_HOST=mysql
DB_PORT=3306
```

Any insights into what might be causing this issue would be appreciated.

### Steps To Reproduce

1. Install Laravel using Sail.
2. Follow the local-only installation steps for Laravel Telescope from the [documentation](https://laravel.com/docs/11.x/telescope#local-only-installation).

**Comment by crynobone** [comment]
Hi there,

Thanks for reporting the problem you are encountering, but it looks like this is a question which may be better suited for a support channel. We only use this issue tracker for reporting bugs with the library itself. If you have a question on how to use functionality provided by this repository you can try one of the following channels:

* [Laracasts Forums](https://laracasts.com/discuss)
* [Laravel.io Forums](https://laravel.io/forum)
* [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
* [Discord](https://discordapp.com/invite/KxwQuKb)
* [Larachat](https://larachat.co/)
* [IRC](https://web.libera.chat/?nick=laravelnewbie&channels=#laravel)

However, this issue will not be locked and everyone is still free to discuss solutions to your problem!

Thanks.

**Comment by Alexrubl** [comment]
Такая же петрушка

**Comment by dimzeta** [comment]
I had been getting this error for months, only after installing Telescope. I couldn't figure out where it was coming from, and it was constantly cluttering up my logs. I spent hours and hours tearing my hair out trying to understand what was going on, and... I found the problem!

I work on VSCode, and I have the **"Laravel Extra Intellisense"** extension, which executes commands every minute outside of Docker (Sail). On my machine, this triggers an Event _(SocialiteWasCalled)_, and Telescope tries to insert it into the DB, still outside of Docker, which causes the famous error `php_network_getaddresses: getaddrinfo for mysql failed: nodename nor servname provided, or not known`.

By configuring Laravel Extra Intellisense correctly, everything went back to normal.

In settings.json, enter:

```json
    "LaravelExtraIntellisense.basePathForCode": "/var/www/html",
    "LaravelExtraIntellisense.phpCommand": "docker-compose exec -w /var/www/html laravel.test php -r \"{code}\""
```

See: https://github.com/amir9480/vscode-laravel-extra-intellisense?tab=readme-ov-file#sample-config-to-use-docker

I feel like I'm alive again. 🥲
